### PR TITLE
Adding a namespace for TinyDB in the companion

### PR DESCRIPTION
**Pull request by ColinTree**
Implement #907

### CODE REVIEW

File: appinventor/appengine/src/com/google/appinventor/client/OdeMessages.java [hunk 1]
```diff
@@ -2663,6 +2663,7 @@ String newerVersionComponentException(String componentType, int srcCompVersion,
   String webViewerComponentPallette();
 
   // Component Properties
+
```
Comment (ewpatton) [inline appinventor/appengine/src/com/google/appinventor/client/OdeMessages.java]: Extraneous whitespace

Comment (ColinTree) [reply appinventor/appengine/src/com/google/appinventor/client/OdeMessages.java]: Done

**Comment by ColinTree** [comment]
I think I need to update html document as well.

**Comment by moliata** [comment]
@ColinTree it would be great!

**Comment by ColinTree** [comment]
@ewpatton I think this may be ready to be reviewed now~

File: appinventor/components/src/com/google/appinventor/components/runtime/TinyDB.java [hunk 2]
```diff
@@ -76,7 +77,19 @@
...
     super(container.$form());
     context = (Context) container.$context();
-    sharedPreferences = context.getSharedPreferences("TinyDB1", Context.MODE_PRIVATE);
+    Namespace(DEFAULT_NAMESPACE);
+  }
+
+  @SimpleProperty(description = "Namespace for storing data.", category = PropertyCategory.BEHAVIOR)
+  @DesignerProperty(editorType = PropertyTypeConstants.PROPERTY_TYPE_TEXT, defaultValue = DEFAULT_NAMESPACE)
+  public void Namespace(String namespace) {
+    this.namespace=namespace;
```
Comment (ewpatton) [inline appinventor/components/src/com/google/appinventor/components/runtime/TinyDB.java]: Add whitespace around `=`

**Comment by ColinTree** [comment]
Done. @ewpatton

**Pull request merged**